packages/okta-auth-js/lib/fetch/fetchRequest.js

```diff
@@ -33,10 +33,15 @@ function fetchRequest(method, url, args) {
     var error = !response.ok;
     var status = response.status;
     var respHandler = function(resp) {
+      var isJSON = typeof resp === 'object'; 
```

Comment:
This feels weird.  It only makes sense when I scroll down and notice this is all defining `respHandler`, which is passed either the raw text, an object from JSON, or an error object.

isJSON is actually the OPPOSITE - it's not JSON, it's an object.  When `isJSON` is true, this calls JSON.stringify() (converting the object to JSON).

I suggest:
1) Defining `respHandler` outside of `fetchRequest` - it distracts 
2) Changing `isJSON` to something else (`isObj`, or perhaps `notYetJSON`)

Reply:
Good feedback! I have refactored for better readability. Please take another look